alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,65 @@
+# AggregatedAPIErrors
+
+## Meaning
+
+This alert is triggered in case multiple calls on the aggregated API of OpenShift failed over a certain time window.
+
+## Impact
+Errors on the aggregated API can result in the unavailability of some OpenShift services.  
+
+## Diagnosis
+
+##### Check the APIs status checks are on True
+
+Currently, there are three aggregated APIs in an OpenShift Cluster. The API on the `openshift-apiserver` namespace, the prometheus-adapter on the namespace `openshift-monitoring`, and the package-server service in the `openshift-operator-lifecycle-manager` namespace.
+
+To get a list of the aggregated APIs and the `APIGroup` to whom they answer requests use the following command:
+
+```shell
+$ oc get apiservice -n openshift-apiserver | grep openshift-
+v1.apps.openshift.io                            openshift-apiserver/api                                      True        340d
+v1.authorization.openshift.io                   openshift-apiserver/api                                      True        340d
+v1.build.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.image.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.oauth.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.packages.operators.coreos.com                openshift-operator-lifecycle-manager/packageserver-service   True        340d
+v1.project.openshift.io                         openshift-apiserver/api                                      True        340d
+v1.quota.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.route.openshift.io                           openshift-apiserver/api                                      True        340d
+v1.security.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.template.openshift.io                        openshift-apiserver/api                                      True        340d
+v1.user.openshift.io                            openshift-apiserver/api                                      True        340d
+v1beta1.metrics.k8s.io                          openshift-monitoring/prometheus-adapter                      True        340d
+```
+Here we want the availability status to be `True`. A `False` means that requests for that API service, API server pods, or resources belonging to that apiGroup failed many times during the last minutes.
```

Comment:
and if you see a `False`, then what?  For example, the following section suggests checking pod logs if a pod is not `Running`.  Presumably in this case you are supposed to dig into pods in the referenced namespace.  Maybe the *Check the `apiserver` pods* suggestion is supposed to be a sub-case of this section, when one of the `openshift-apiserver/api` entries is `False`?